qds_sdk/account.py

```diff
@@ -43,6 +43,15 @@ def parsers():
             choices=["true", "false"], default="false",
             help="Use previous account plan, default: false")
         create.set_defaults(func=AccountCmdLine.create)
+
+        #branding
+        branding = subparsers.add_parser(
```

Comment:
1. yes in api it is already checking if uri exist or not for a given account.
2.making a blank call with just account id will return status "Enter either logo or link parameters" from api.
3. label of documentation url is name of the link which user want to give.Should I change it to name of link-url ?